mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -199,6 +202,37 @@ void registerGroupShouldCreateRetryExchange() throws Exception {
             .anyMatch(exchange -> exchange.getName().equals(retryExchangeName.asString()));
     }
 
+    @Nested
+    class AvailabilityTest {
+
+        @Test
+        void inProcessingEventShouldBeReDispatchedToAnotherEventBusWhenOneIsDown() throws Exception {
```

Comment:
this case test a stopped EventBus. But It doesn't reflect 100% a crashed EventBus or Networking issue case.